README.md

```diff
@@ -6,10 +6,11 @@ A [Swagger](https://swagger.io/) API client generator based on [axios](https://g
 
 ## Usage
 
-If you pass a [Swagger definition](https://swagger.io/docs/specification/2-0/basic-structure/) (v2.0; valid JSON) to Swaxios, then it will generate you an API client that uses axios under the hood and is written in TypeScript.
+If you pass an [OpenAPI definition](https://swagger.io/docs/specification/2-0/basic-structure/) file (v2.0; valid JSON or yaml) to Swaxios, then it will generate you an API client that uses axios under the hood and is written in TypeScript.
```

Comment:
```suggestion
If you pass an [OpenAPI definition](https://swagger.io/docs/specification/2-0/basic-structure/) file (v2.0; valid JSON or YAML) to Swaxios, then it will generate you an API client that uses axios under the hood and is written in TypeScript.
```